### ISSUE TITLE
Save from Page-Designer directly to VSCode

### ISSUE THREAD

**Issue by luxferresum** [Microsoft/AL#110]
It would be nice to have a workflow where changes from the Page-Designer could direclty be saved back to VSCode. The current workflow requires to
- download the app as .zip
- extract the `.al` file
- copy it to the Code-Project.

Usually one big benefit of the new concept would be that it would be easy to use git for source control. The In-Page-Designer sadly works a bit against this. A way to edit an existing project on the file system would be awesome, ideally by a VSCode extension to install the extionsion, edit it with the In-Page-Designer and save it back to the `.al` files.

**Comment by StanislawStempin** [comment]
I agree with your suggestion, we are currently working on a tighter integration between the Designer and Code. We envision a "pull" command from VS Code that will get the latest source code for a given project from the server. 
Longer term we might look into hosting the Designer locally or within VS Code and to allow direct edit of files on disk but there are several technical constraints to solve before we can do this.
